makefile

```diff
@@ -5,10 +5,10 @@ CFLAGS += $(EXTRA_FLAGS)
 
 LDFLAGS =
 ifeq ($(UNAME), Linux)
-	LDFLAGS += -L/usr/lib/x86_64-linux-gnu -lboost_system -lboost_filesystem -lpthread
+	LDFLAGS += -L/usr/lib/x86_64-linux-gnu -static-libgcc -static-libstdc++ -lpthread -Wl,-Bstatic -lboost_system -lboost_filesystem
```

Comment:
We may only want to statically link when we're deploying, instead of whenever we build on linux. I think we could do that by adding an extra variable when we compile using docker. But let's look into that after we get this working